buildkitd/buildkitd.pprof.template

```diff
@@ -0,0 +1 @@
+  debugAddress = "0.0.0.0:6060"
```

Comment:
nit: missing newline.

Reply:
I can add it. It just s p r e a d s out the config file in our logs :(

Reply:
maybe `head -c -1 buildkitd.pprof.template` could be used instead?

but that makes things brittle too :(